contracts/services/QueryService.sol

```diff
@@ -111,6 +111,12 @@ interface IQueryService is IVersioned {
         address[] memory factories,
         INameRegistry nameRegistry
     ) external view returns (Structs.AssetCommonStateWithName[] memory);
+
+    function getAssetStatesForIssuer(
```

Comment:
```suggestion
    function getAssetsWtihStateForIssuer(
```

`getAssetStatesForIssuer` sounds like you are trying to fetch to different states for assets.